src/components/OrderDataTable.css

```diff
@@ -0,0 +1,513 @@
+body {
```

Comment:
If you end up up reusing these styles for inventory table then this file should be renamed. If there are different styles for inventory and order table then there should also be two separate css files, one for each table